src/Objs/Homepage/HomepageEditingConfig.js

```diff
@@ -95,4 +95,18 @@ Scrivito.provideEditingConfig("Homepage", {
     ...metadataInitialContent,
     showAsLandingPage: "no",
   },
+  validations: [
+    [
+      "title",
```

Comment:
The `title` attribute is defined via `defaultPageEditingConfigAttributes`. can you create something similar for the validations? So Homepage, LandingPage and Page all use the same validation for title?